fastlane/Fastfile

```diff
@@ -16,55 +16,58 @@
-    take_screenshots
+    take_screenshots(options)
   end
 
   desc "Capture screenshots"
   lane :take_screenshots do |options|
-    devs = adb_devices()
-    check_devices(devs: devs)
-    
-    device = devs[0]
-    demo_mode_enter(device_serial: device.serial)
-
-    capture_android_screenshots(app_apk_path: "WordPress/build/outputs/apk/vanilla/debug/WordPress-vanilla-debug.apk",
-        tests_apk_path: "WordPress/build/outputs/apk/androidTest/vanilla/debug/WordPress-vanilla-debug-androidTest.apk", 
-        use_tests_in_classes: "org.wordpress.android.ui.screenshots.WPScreenshotTest", 
-        reinstall_app: true,
-        device_type: options[:device],
-        locales: ["ar", "de-DE", "en-US", "es-ES", "fr-CA", "fr-FR", "id", "it-IT", "iw-IL", "ja-JP", "ko-KR", "nl-NL", "pl-PL", "pt-BR", "ru-RU", "sr", "sv-SE", "th", "tr-TR", "vi", "zh-CN", "zh-TW"]
-        )
-        
-    demo_mode_exit(device_serial: device.serial)
-  end
-
-  # Screenshot helpers
-  private_lane :check_devices do |options|
-    devs = options[:devs]
-    if devs.count == 0 then
-      UI.user_error!("No devices detected. Please, connect one device")
-    end
-    
-    if devs.count > 1 then
-      UI.user_error!("Too many devices detected. Please, connect one device")
-    end
-  end
+    screenshot_devices = [
+      {
+        screenshot_type: 'phone',
+        device_name: 'fastlane_screenshots_phone',
+        device_serial: options[:phone_serial],
+      },
+      {
+        screenshot_type: 'sevenInch',
+        device_name: 'fastlane_screenshots_seven_inch',
+        device_serial: options[:sevenInch_serial],
+      },
+      {
+        screenshot_type: 'tenInch',
+        device_name: 'fastlane_screenshots_ten_inch',
+        device_serial: options[:tenInch_serial],
+      }
+    ]
 
-  private_lane :demo_mode_enter do |options|
-    # Setup device for demo
-    adb(command: "shell settings put global sysui_demo_allowed 1", serial: options[:device_serial])
-    # Disable system notifications
-    adb(command: "shell am broadcast -a com.android.systemui.demo -e command notifications -e visible false", serial: options[:device_serial])
-    # Enjoy lunch while browsing the Play Store
-    adb(command: "shell am broadcast -a com.android.systemui.demo -e command clock -e hhmm 1337", serial: options[:device_serial])
-  end
+    screenshot_options = {
+      app_apk_path: "WordPress/build/outputs/apk/vanilla/debug/WordPress-vanilla-debug.apk",
+      tests_apk_path: "WordPress/build/outputs/apk/androidTest/vanilla/debug/WordPress-vanilla-debug-androidTest.apk",
+      use_tests_in_classes: "org.wordpress.android.ui.screenshots.WPScreenshotTest",
+      reinstall_app: false,
+      clear_previous_screenshots: true,
+      locales: ["ar", "de-DE", "en-US", "es-ES", "fr-CA", "fr-FR", "id", "it-IT", "iw-IL", "ja-JP", "ko-KR", "nl-NL", "pl-PL", "pt-BR", "ru-RU", "sr", "sv-SE", "th", "tr-TR", "vi", "zh-CN", "zh-TW"]
```

Comment:
We have the list of locales in two different places in the same file.
Would it be possible to have a single shared definition?

Reply:
Yes, absolutely! Was thinking about doing this later but no time like the present 😄